### PR TITLE
refactor: replace inquirer with amplify-prompts in amplify-category-analytics package

### PR DESCRIPTION
**Pull request by kuhlmanp**
refactor: replace inquirer with amplify-prompts in amplify-category-analytics package

Converts references to inquirer with amplify-prompts. Specifically, removes automated code designed to handle tens of questions with simple direct questions since there were only 2 questions in each analytics category. Corresponding metadata in supported-services.json have been removed.

#### Description of how you validated changes

Manual test locally via cli and examination of output json files.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-analytics/src/index.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { $TSContext, $TSAny, AmplifyFault } from 'amplify-cli-core';
 import * as path from 'path';
-import inquirer, { QuestionCollection } from 'inquirer';
```
Comment (lazpavel) [inline packages/amplify-category-analytics/src/index.ts:3]: don't see changes in package.json, we need to remove the references for inquirer from dependencies section as well

File: packages/amplify-category-analytics/src/index.ts [hunk 2]
```diff
@@ -30,16 +29,7 @@ export const console = async (context: $TSContext): Promise<void> => {
...
-      name: 'resource',
-      message: 'Select resource',
-      type: 'list',
-      choices: ['kinesis', 'pinpoint'],
-      required: true,
-    };
-
-    const result = await inquirer.prompt(question as QuestionCollection<{ [x: string]: unknown }>);
-    selectedResource = result.resource;
+    selectedResource = await prompter.pick('Select resource', ['kinesis', 'pinpoint']);
```
Comment (lazpavel) [inline packages/amplify-category-analytics/src/index.ts:32]: I am not sure what required means here? if we didn't allow empty selection before we might need to set the `pickAtLeast: 1` option. Can you please confirm this?

**Review by lazpavel — COMMENTED** [review]
Please trigger the cloud-e2e. Can you confirm that the affected prompts are covered by existing unit or e2e tests? if not can you add?

File: packages/amplify-category-analytics/src/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.ts [hunk 3]
```diff
@@ -67,112 +58,87 @@ const configure = (
...
-      question = {
-        type: 'input',
-        ...question,
-      };
-    }
-    questions.push(question);
+  const answers = {
+    resourceName:
+      resourceName ||
+      (await prompter.input('Provide a friendly resourcename:', {
```
Comment (edwardfoyle) [inline packages/amplify-category-analytics/src/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.ts:64]:
```suggestion
      (await prompter.input('Provide a friendly resource name:', {
```

File: packages/amplify-category-analytics/src/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.ts [hunk 4]
```diff
@@ -67,112 +58,87 @@ const configure = (
...
+        validate: alphanumeric('Resource name should be alphanumeric'),
```
Comment (edwardfoyle) [inline packages/amplify-category-analytics/src/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.ts:65]:
```suggestion
        validate: alphanumeric('Resource name must be alphanumeric'),
```

File: packages/amplify-category-analytics/src/index.ts (hunk 2 above)
Comment (kuhlmanp) [reply packages/amplify-category-analytics/src/index.ts:32]: When testing I didn't see an obvious way to bypass this selection. I checked the prompter and inquirer and the default behavior is to require a choice when prompted in the pick one scenario.

File: packages/amplify-category-analytics/src/index.ts (hunk 1 above)
Comment (kuhlmanp) [reply packages/amplify-category-analytics/src/index.ts:3]: Yep, will do that now.

**Review by evcodes — APPROVED** [review]
LGTM

**Comment by kuhlmanp** [comment]
all e2e tests appear to be passing now except for the integration-test, which doesn't look related to my changes.

**Review by lazpavel — APPROVED** [review]
LGTM

**Review by evcodes — APPROVED** [review]

**Pull request merged**